web/packages/shared/components/Notification/Notification.story.tsx

```diff
@@ -149,7 +155,109 @@ export const Notifications = () => {
           isAutoRemovable={false}
         />
       </Flex>
-    </Flex>
+
+      <Flex flexDirection="column" gap={4}>
+        <Notification
+          item={{
+            id: crypto.randomUUID(),
+            severity: 'info',
+            content: {
+              title: 'Info with link',
+              description: loremIpsum,
+              link: {
+                href: 'https://goteleport.com',
+                text: 'goteleport.com',
+              },
+            },
+          }}
+          Icon={Info}
+          getColor={theme => theme.colors.info}
+          onRemove={() => {}}
+          isAutoRemovable={false}
+        />
+        <Notification
+          item={{
+            id: crypto.randomUUID(),
+            severity: 'warn',
+            content: {
+              title: 'Warning with link',
+              description: loremIpsum,
+              link: {
+                href: 'https://goteleport.com',
+                text: 'goteleport.com',
+              },
+            },
+          }}
+          Icon={Warning}
+          getColor={theme => theme.colors.warning}
+          onRemove={() => {}}
+          isAutoRemovable={false}
+        />
+        <Notification
+          item={{
+            id: crypto.randomUUID(),
+            severity: 'error',
+            content: {
+              title: 'Error with link',
+              description: loremIpsum,
+              link: {
+                href: 'https://goteleport.com',
+                text: 'goteleport.com',
+              },
+            },
+          }}
+          Icon={Warning}
+          getColor={theme => theme.colors.danger}
+          onRemove={() => {}}
+          isAutoRemovable={false}
+        />
+      </Flex>
+
+      <Flex flexDirection="column" gap={4}>
+        <Notification
+          item={{
+            id: crypto.randomUUID(),
+            severity: 'info',
+            content: {
+              title: 'Info with list',
+              list: [loremIpsum, loremIpsum],
```

Comment:
Examples of notifications with lists seem to not render correctly for me. 🤔

![Notifications with lists](https://user-images.githubusercontent.com/27113/222179395-adf138b8-b906-4213-a80c-be1ae4ae422e.png)

Reply:
Ah that's because of the type guard which was checking if `description !== undefined`. Later I changed `description` to be optional and forgot to update it. 
It is unnecessary and I removed it completely.